simapp/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.23.4
 
 require (
 	cosmossdk.io/api v0.8.2 // main
-	cosmossdk.io/client/v2 v2.10.0-beta.1
+	cosmossdk.io/client/v2 v2.10.0-beta.2 // main
```

Comment:
this is actually tag form release/v0.52.x due to the sdk dep of client/v2

Reply:
so, we should change the comment here right?

Reply:
Yeah no need to append // main